src/components/accounts/GenerateAccount/DownloadDialog/downloadDialog.js

```diff
@@ -16,13 +16,13 @@ const DownloadDialog = (props) => {
                 <div style={formStyles.right}>
                     <div>
                         <div className={ styles.title }>
-                            Download the Wallet File
+                            Download the Account File
                         </div>
                         <div className={ styles.subTitle }>
                             And save the copy in a safe place (not on this computer).
                         </div>
                         <div className={ styles.description }>
-                            You need a Wallet File to make all operations with account. When you want to view your balance or send something from your Ethereum Wallet, you will need this file.  You will also need the strong password you created earlier.
+                            You need an Account File to make all operations with account. When you want to view your balance or send something from your Ethereum Classic Account, you will need this file.  You will also need the strong password you created earlier.
```

Comment:
Would it make more sense to say "Account _Keyfile_"? Or "Account Key File"? I'm getting kind of nitpicky here, but IMO "Account File" conflates the idea of an account with idea of maintaining _access to_ that account.

Reply:
yea, what it _really_ is, is adding a key to a keychain

Reply:
I also suggest ~view your balance or send something from~ --> "_access your account or create a transaction_. Account keys are not required to view balances, or?